lib/PuppeteerSharp/LauncherBase.cs

```diff
@@ -146,13 +146,19 @@ public async Task<bool> WaitForExitAsync(TimeSpan? timeout)
             return true;
         }
 
+        /// <summary>
+        /// Disposes Base process and any temporary user directory.
+        /// </summary>
+        /// <param name="disposing">Indicates whether disposal was initiated by <see cref="Dispose()"/> operation.</param>
+        protected virtual void Dispose(bool disposing) => _stateManager.CurrentState.Dispose(this);
+
         /// <summary>
         /// Set Env Variables.
         /// </summary>
         /// <param name="environment">The environment.</param>
         /// <param name="customEnv">The customEnv.</param>
         /// <param name="realEnv">The realEnv.</param>
-        protected static void SetEnvVariables(IDictionary<string, string> environment, IDictionary<string, string> customEnv, IDictionary realEnv)
+        private static void SetEnvVariables(IDictionary<string, string> environment, IDictionary<string, string> customEnv, IDictionary realEnv)
```

Comment:
FYI: This is also a breaking change

Reply:
hahaha fair. I have to `seal` some classes.

Reply:
changed